docker/build_scripts/build_utils.sh

```diff
@@ -26,12 +26,18 @@ function lex_pyver {
 function do_python_build {
     local py_ver=$1
     check_var $py_ver
-    mkdir -p /opt/$py_ver/lib
+    local soabi_flags=$2
+    check_var $soabi_flags
+    mkdir -p /opt/${py_ver}${soabi_flags}/lib
     if [ $(lex_pyver $py_ver) -lt $(lex_pyver 3.3) ]; then
-        local unicode_flags="--enable-unicode=ucs4"
+        if [ $soabi_flags = "mu" ]; then
+            local unicode_flags="--enable-unicode=ucs4"
+        else
+            local unicode_flags="--enable-unicode=ucs2"
+        fi
     fi
     # -Wformat added for https://bugs.python.org/issue17547 on Python 2.6
-    CFLAGS="-Wformat" LDFLAGS="-Wl,-rpath /opt/$py_ver/lib" ./configure --prefix=/opt/$py_ver --disable-shared $unicode_flags > /dev/null
+    CFLAGS="-Wformat" LDFLAGS="-Wl,-rpath /opt/${py_ver}${soabi_flags}/lib" ./configure --prefix=/opt/${py_ver}${soabi_flags} --disable-shared $unicode_flags > /dev/null
```

Comment:
Makes sense to me, there should be no reason we need an rpath pointing at the Python lib directory without a shared libpython.